services/movie.py

```diff
@@ -0,0 +1,43 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
+) -> QuerySet(Movie):
+    queryset = Movie.objects.all()

```

Comment:
Please simplify this method similarly to how you did it in the 'create_movie' method.
```
if genres_ids:
        queryset = queryset.filter(genres__id__in=genres_ids)
```
        
and so on...